pkg/pillar/types/errortime.go

```diff
@@ -12,19 +12,95 @@ import (
 
 // Common error with timestamp
```

Comment:
This comment has become detached from what it is commenting on, and the new struct doesn't have an explanatory comment.